addons/binding/org.openhab.binding.tankerkoenig/src/main/java/org/openhab/binding/tankerkoenig/handler/WebserviceHandler.java

```diff
@@ -139,6 +139,13 @@ public void updateStationData() {
                 // if the result is not OK, no updates are done and the status of the Bridge goes to OFFLINE!
                 updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR,
                         "Empty return or no internet connection");
+                // if the Bridge goes OFFLINE, all Stations will go OFFLINE as well
+                for (Thing thing : getThing().getThings()) {
+                    StationHandler tkh = (StationHandler) thing.getHandler();
+                    tkh.updateStatus(ThingStatus.OFFLINE);
```

Comment:
Does ESH have no smarter way to do this, will they not already be offline if the bridge is offline, or is there maybe a hook which gets called when the bridge changes status?